From 51a6e1f30d9ee94cfcf5d60d1bfdeff295b91c16 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Fri, 29 Sep 2023 17:01:01 +0200
Subject: [PATCH 0043/1695] plat-stm32mp1: conf: enable support for GPIO
 regulators

Enables support for GPIO regulators on platform stm32mp1 when
CFG_STM32_GPIO is enabled.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 0c59f6b9c864cf403220953fb38e359d0f02de66 plat-stm32mp1: conf: enable support for GPIO regulators]
Change-Id: I6110be9be4c201fa9afd9012d5277d5447d3582e
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from fbf57d28d3cbcf6e3a79390803890b240c12d27a]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index bf1d59942..90f6c853b 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -210,6 +210,7 @@ CFG_STPMIC1 ?= y
 CFG_TZC400 ?= y
 
 CFG_DRIVERS_I2C ?= $(CFG_STM32_I2C)
+CFG_REGULATOR_GPIO ?= $(CFG_STM32_GPIO)
 
 CFG_WITH_SOFTWARE_PRNG ?= n
 ifneq ($(CFG_WITH_SOFTWARE_PRNG),y)
-- 
2.47.2

